### PR TITLE
the main structure

### PR DESCRIPTION
**Pull request by ahmadmhd357**

### CODE REVIEW

**Review by batoulst — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-redux": "^8.0.1",
+    "react-router-dom": "^6.5.0",
```
Comment (Omer-Amr) [inline package.json:14]: Great to make the installation of the react router "with the latest version" which is necessary for routing inside our website!

File: src/App.jsx [hunk 2]
```diff
@@ -1,10 +1,21 @@
...
+import About from './components/About';
+import Blog from './components/Blog';
+import Contacts from './components/Contacts';
+import Home from './components/Home';
+import Navbar from './components/Navbar';
 
 function App() {
   return (
     <div className="App">
-     <h1>Test test</h1>
```
Comment (Omer-Amr) [inline src/App.jsx:7]: Nice one to make the very first line in our website :)

File: src/App.jsx [hunk 3]
```diff
@@ -1,10 +1,21 @@
 import React from 'react';
-
+import {Routes, Route} from 'react-router-dom'
+import About from './components/About';
+import Blog from './components/Blog';
+import Contacts from './components/Contacts';
+import Home from './components/Home';
+import Navbar from './components/Navbar';
```
Comment (Omer-Amr) [inline src/App.jsx:8]: Great job Ahmad!

File: src/index.jsx [hunk 4]
```diff
@@ -1,13 +1,15 @@
...
+import { BrowserRouter } from 'react-router-dom';
 import App from './App';
 import './index.css';
 
 const container = document.getElementById('root');
 const root = createRoot(container);
 
 root.render(
-
-      <App />
```
Comment (Omer-Amr) [inline src/index.jsx:11]: I would like to mention the opening "App" should be like this one <App> , it's the vs code autocorrection 😉

**Review by Omer-Amr — COMMENTED** [review]
I appreciate your efforts!

**Review by Omer-Amr — APPROVED** [review]

**Pull request merged**